docker-compose.yaml

```diff
@@ -7,14 +7,21 @@ services:
       - 8080:8080
       - 5001:5001
   vitalam-node:
-    image: ghcr.io/digicatapult/vitalam-node:v2.4.0
+    image: ghcr.io/digicatapult/vitalam-node:v2.5.0
+    command: /bin/sh -c "
```

Comment:
~I believe the `entrypoint` for the container is already set to `vitalam-node`~

Scratch that it's done using `CMD`. We should update that at some point which is a little annoying